src/main/java/pl/numerki/frontend/ChartGenerator.java

```diff
@@ -0,0 +1,46 @@
+package pl.numerki.frontend;
+
+import org.jfree.chart.ChartFactory;
+import org.jfree.chart.JFreeChart;
+import org.jfree.chart.plot.PlotOrientation;
+import org.jfree.chart.plot.XYPlot;
+import org.jfree.chart.renderer.xy.XYLineAndShapeRenderer;
+import org.jfree.data.xy.XYSeries;
+import org.jfree.data.xy.XYSeriesCollection;
+
+import java.util.function.Function;
+
+public class ChartGenerator {
+    public static JFreeChart generatePlot(
+            Function<Double, Double> function, double leftCompartment, double rightCompartment,
+            double bisectionZeroPosition, double secantZeroPosition
+    ) {
+        XYSeries series = new XYSeries(0);
+        double unit = (rightCompartment - leftCompartment) / 1000;
```

Comment:
Shouldn't the user be able to choose number of points ?

Reply:
hmmm, good question